tests/nestedfor.c

```diff
@@ -0,0 +1,11 @@
+#include<stdio.h>
```

Comment:
Maybe we can change `for.c` to nested for loop. As long as the nested for loop is correct, single for loop should be correct too.